baseapp/baseapp/content_feed/migrations/0001_initial.py

```diff
@@ -0,0 +1,69 @@
+# Generated by Django 5.0.6 on 2024-11-27 21:51
+import django.db.models.deletion
+import django.utils.timezone
+import model_utils.fields
+import swapper
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        migrations.swappable_dependency(settings.BASEAPP_PROFILES_PROFILE_MODEL),
+        swapper.dependency("baseapp_content_feed", "ContentPost"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="ContentPost",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True, primary_key=True, serialize=False, verbose_name="ID"
+                    ),
+                ),
+                (
+                    "created",
+                    model_utils.fields.AutoCreatedField(
+                        default=django.utils.timezone.now, editable=False, verbose_name="created"
+                    ),
+                ),
+                (
+                    "modified",
+                    model_utils.fields.AutoLastModifiedField(
+                        default=django.utils.timezone.now, editable=False, verbose_name="modified"
+                    ),
+                ),
+                ("content", models.TextField()),
+                (
+                    "profile",
+                    models.ForeignKey(
+                        blank=True,
+                        null=True,
+                        on_delete=django.db.models.deletion.SET_NULL,
+                        related_name="content_posts",
+                        to=settings.BASEAPP_PROFILES_PROFILE_MODEL,
```

Comment:
here as well 
```suggestion
                        to=swapper.get_model_name("baseapp_profiles", "Profile"),
```

those are important because `settings.BASEAPP_PROFILES_PROFILE_MODEL` can be undefined